crm_request/models/request.py

```diff
@@ -252,10 +249,18 @@ def write(self, values):
         """
         super().write(values)
 
-        if values.get("stage_id") == self.env.ref("crm_request.stage_wait_support").id:
+        # Moving a request to waiting for support
+        # automaticaly assign it to yourself
+        if values.get("stage_id") == self.env.ref(
+                "crm_request.stage_wait_support").id:
```

Comment:
Also  in state resolve. please

Reply:
there is no ref to the state resolve, should I had it the crm_request_data.xml ?